cherryontech/JourneyWise
# Archetype

## Pull Request

### Description
Added the archetypes that the user can read.  

### Related Issue
worked on the issue https://marisolsaumi.atlassian.net/browse/JW-50

### Description of Changes
The user can navigate to the different archetypes 
### Checklist
- [x ] I have read the CONTRIBUTING.md document.
- [ x] My code follows the code style of this project.
- [ x] My changes generate no new warnings or errors.
- [x ] I have updated the documentation or comments where necessary.
- [ x] All tests pass locally.

### Testing Instructions
Do a pull request of the branch and run it on your local machine to see that it works properly. 
### Screenshots (if applicable)
<img width="407" alt="Screenshot 2024-05-12 at 21 28 14" src="https://github.com/cherryontech/JourneyWise/assets/77724559/4157843c-a48c-4e5b-8f02-c797889e370b">

### Gotchas/Learnings
Doing mobile first approach

### Additional Notes (Optional)
* Include any other relevant information for reviewers.
